.circleci/config.yml

```diff
@@ -274,6 +278,12 @@ jobs:
             export RECORDS_MOVER_SESSION_TYPE=env
             mkdir -p test-reports/itest
             cd tests/integration/records/single_db
+            # TODO: Apply same pattern in with_dbs version
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker